docs/user-guide.md

```diff
@@ -2069,6 +2069,14 @@ You can apply Oracle Release Update (RU) or Patch Set Update (PSU) patches to
 both the Grid Infrastructure and Database homes by using the
 `apply-patch.sh` script of the toolkit.
 
+By default, `install-oracle.sh` updates to the latest available patch.  To
+apply a specific patch instead, use `install-oracle.sh`'s `--no-patch` option 
+to skip patching at installation time, and then execute `apply-patch.sh` with
```

Comment:
"...at installation time. After installation is complete, execute the `apply-patch.sh` script with..."